src/tests/ftest/container/snapshot.py

```diff
@@ -206,7 +206,7 @@ def test_snapshot_negativecases(self):
 
         Use Cases: Combinations with minimun 1 client and 1 server.
         :avocado: tags=all,small,smoke,pr,snap,snapshot_negative,
-                       snapshotcreate_negative
+        :avocado: tags=snapshotcreate_negative
```

Comment:
Interesting.  I've never considered doing that.  Does it work?  Not that I'd be surprised.